dbt/models/qc/docs.md

```diff
@@ -128,61 +147,32 @@ Pulled from the following Inquire queries:
 - `FP Checklist - Non-EX, RR parcels with 0 value` (#1570) - Will C.
 {% enddocs %}
 
-# vw_report_town_close_500k_increase_1m_decrease
-
-{% docs view_vw_report_town_close_500k_increase_1m_decrease %}
-Check for rows in the  `ASMT` table where the total market value has increased
-by more than $500k or decreased by more than $1m in the past year.
-
-This view is exported as part of the QC report to check values prior to town
-closings.
-
-Pulled from the following Inquire queries:
-
-- `FP Checklist - 500k increase, 1m decrease` (#1673) - Will C.
-- `FP Checklist - Card Code Info` (#2160) - Will C.
-{% enddocs %}
-
-# vw_report_town_close_class_does_not_equal_luc
-
-{% docs view_vw_report_town_close_class_does_not_equal_luc %}
-Check for parcels where the class does not match the land use code in
-the `PARDAT` table.
-
-This view is exported as part of the QC report to check values prior to town
-closings.
-
-Pulled from the following Inquire queries:
-
-- `FP Checklist - Class does not equal LUC` (#2046) - Will C.
-{% enddocs %}
-
-# vw_report_town_close_res_parcels_not_set_to_cost_approach
+# vw_report_town_close_289s
 
-{% docs view_vw_report_town_close_res_parcels_not_set_to_cost_approach %}
-Check for residential class parcels that are using a cost approach.
+{% docs view_vw_report_town_close_289s %}
+Check for rows in the  `OBY` table where the class is 289.
 
 This view is exported as part of the QC report to check values prior to town
 closings.
 
 Pulled from the following Inquire queries:
 
-- `FP Checklist - Res parcels not set to Cost Approach` (#2114) - Will C.
+- `FP Checklist - 289s` (#1963) - Will C.
 {% enddocs %}
 
-# vw_report_town_close_ovrrcnlds_to_review
+# vw_report_town_close_500k_increase_1m_decrease
 
-{% docs view_vw_report_town_close_ovrrcnlds_to_review %}
-Check for rows in the `DWELDAT`, `COMDAT`, and `OBY` tables
-where the calculated net market value does not match the override net market
-value.
+{% docs view_vw_report_town_close_500k_increase_1m_decrease %}
+Check for rows in the  `ASMT` table where the total market value has increased
```

Comment:
```suggestion
Check for rows in the `ASMT` table where the total market value has increased
```

Reply:
Oops, good catch! Fixed along with another stray whitespace in da946cc145e0b5c1f82d0dcb5075c957110734d4.